packages/page/src/Application/MessageHandler/CopyPageMessageHandler.php

```diff
@@ -32,25 +33,26 @@ public function __construct(
     ) {
     }
 
-    public function __invoke(CopyPageMessage $message): void
+    public function __invoke(CopyPageMessage $message): PageInterface
     {
-        $page = $this->pageRepository->getOneBy($message->getIdentifier());
-        $targetParentPage = $this->pageRepository->getOneBy($message->getDestinationIdentifier());
+        $sourcePage = $this->pageRepository->getOneBy($message->getSourceIdentifier());
+        $targetParentPage = $this->pageRepository->getOneBy($message->getTargetParentIdentifier());
 
         $targetPage = $this->pageRepository->createNew();
```

Comment:
@Prokyonn is not here the uuid missing:


```suggestion
        $targetPage = $this->pageRepository->createNew($message->getTargetUuid());
```